src/features/plans/pages/PlanDetailPage.tsx

```
@@ -0,0 +1,198 @@
+          <div className="relative">
+            <button
+              onClick={() => setIsManageOpen(prev => !prev)}
+              className="rounded-md border border-border/50 px-3 py-1.5 text-sm hover:bg-secondary"
+            >
+              관리
+            </button>
+
+            {isManageOpen && (
+              <>
+                <div className="fixed inset-0 z-40" onClick={() => setIsManageOpen(false)} />
+                <div className="absolute right-0 z-50 mt-2 w-36 rounded-md border border-border bg-background shadow-lg">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)